CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Changelog
 
-## ✨ yew **0.20.0** *(2022-11-xx)*
+## ✨ yew **0.20.1** *(2023-01-09)* Changelog
+
+### 🛠 Fixes
+
+- Fix top-level reconciliation in portals. [[@WorldSEnder](https://github.com/WorldSEnder), [#3020](https://github.com/yewstack/yew/pull/3020)]
+- Fix clippy::let_unit_value lint in propless components. [[@WorldSEnder](https://github.com/WorldSEnder), [#2970](https://github.com/yewstack/yew/pull/2970)]
+- Reentrant event listeners. [[@WorldSEnder](https://github.com/WorldSEnder), [#3037](https://github.com/yewstack/yew/pull/3037)]
+- Assert there are no circular references. [[@WorldSEnder](https://github.com/WorldSEnder), [#3025](https://github.com/yewstack/yew/pull/3025)]
+  This fixes a performance regression in yew 0.20 with large lists.
+
+### ⚡️ Features
+
+- Add method map() on Children to wrap easily. [[@Cecile Tonglet](https://github.com/Cecile Tonglet), [#3039](https://github.com/yewstack/yew/pull/3039)]
+- Add impl IntoIterator on &Classes. [[@Cecile Tonglet](https://github.com/Cecile Tonglet), [#3038](https://github.com/yewstack/yew/pull/3038)]
+- Make Classes cheap to clone. [[@Cecile Tonglet](https://github.com/Cecile Tonglet), [#3021](https://github.com/yewstack/yew/pull/3021)]
```

Comment:
This is actually a breaking change. This makes Classes !Send, which users have already mentioned on Discord that it breaks their code.